tasks.py

```diff
@@ -799,6 +799,47 @@ def integration_test(
     )
 
 
+@task(
+    help={
+        "dataset": "File (.sql.tar.gz) to start from that will untar to 'nautobot.sql'",
+        "db_engine": "mysql or postgres",
```

Comment:
Instead of requiring `--db_engine` could we introspect the output of either `docker-compose config --format json` or `docker-compose config --images`? In the json format you could look at `json["services"]["db"]["image"]`:

```json
{
  "name": "nautobot",
  "networks": {
    "default": {
      "name": "nautobot_default",
      "ipam": {},
      "external": false
    }
  },
  "services": {
  -- snipped
    "db": {
      "command": null,
      "entrypoint": null,
      "environment": {
        -- snipped
      },
      "healthcheck": {
        "test": [
          "CMD-SHELL",
          "pg_isready"
        ],
        "timeout": "5s",
        "interval": "5s",
        "retries": 50
      },
      "image": "postgres:13",
      "networks": {
        "default": null
      },
      "volumes": [
        {
          "type": "volume",
          "source": "pgdata_nautobot",
          "target": "/var/lib/postgresql/data",
          "volume": {}
        }
      ]
    },
    -- snipped
}
```

The `--images` output is simpler but perhaps more fragile:

```
local/nautobot-dev:local-py
local/nautobot-dev:local-py
postgres:13
local/nautobot-dev:local-py
node:18-slim
redis:6-alpine
selenium/standalone-firefox:4.1.2
```

Reply:
That'll work for the local docker workflow, but won't it be a problem for non-docker workflows and for CI?

Reply:
Yeah :disappointed:. I always forget about the non docker workflows.